wa/commands/show.py

```diff
@@ -37,17 +39,22 @@ def execute(self, state, args):
         name = args.plugin
         rst_output = None
 
-        plugin = get_plugin(name)
-        if plugin:
-            rst_output = get_rst_from_plugin(plugin)
-            plugin_name = plugin.name
-            kind = '{}:'.format(plugin.kind)
+        if name == caseless_string('settings'):
```

Comment:
In addition to documenting global settings here, I would also add a separate section (still under `settings` keyword) for environment variables.